sql/src/main/java/io/druid/sql/calcite/expression/builtin/CeilOperatorConversion.java

```diff
@@ -80,7 +80,7 @@ public DruidExpression toDruidExpression(
       // So there is no simple extraction for this operator.
       return DruidExpression.fromFunctionCall(
           "timestamp_ceil",
-          ImmutableList.of(
+          Arrays.asList(
```

Comment:
Why this change?

Reply:
Maybe `Stream.of()` is better.

Reply:
Changed because `ImmutableList` doesn't support nulls, I suppose. Anyway, a parallel PR #5980 handles this in the whole codebase.